api/tests/Api/TopBooksTest.php

```diff
@@ -9,14 +9,19 @@
 use App\Entity\TopBook;
 use App\State\TopBookCollectionProvider;
 use App\State\TopBookItemProvider;
+use App\Tests\Fixtures\Story\DefaultBooksStory;
 use Symfony\Component\HttpFoundation\Response;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
 
 /**
  * These tests are read only, thus there is not need to use the RefreshDatabaseTrait
  * like BooksTests and ReviewsTest.
  */
 class TopBooksTest extends ApiTestCase
 {
+    use ResetDatabase, Factories;
```

Comment:
PSR-12 recommends to include one-per-line for use traits: https://www.php-fig.org/psr/psr-12/#42-using-traits